docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+version: "3.9"
+
+volumes:
+  app-logs:
+
+services:
+  am:
+    image: autometrics/am:latest
+    extra_hosts:
+      - host.docker.internal:host-gateway
+    ports:
+      - "6789:6789"
+      - "9090:9090"
+    container_name: am
+    command: "start http://otel-collector:9464/metrics"
```

Comment:
i changed the start command to reference the internal network here, not sure if that breaks any other examples